src/test/java/redis/clients/jedis/tests/commands/AllKindOfValuesCommandsTest.java

```diff
@@ -356,6 +356,46 @@ public void ttl() {
   }
 
   @Test
+  public void touch() {
```

Comment:
What about adding an `OBJECT IDLETIME <key>` call in some of the examples to validate that Jedis is actually running the `touch` command?. This way we can verify that the access time of the specified key(s) was actually changed.